### PR TITLE
Add Qt application and QtVideoOp.

### PR DESCRIPTION
**Pull request by AndreasHeumann**

### CODE REVIEW

File: applications/qt/main.cpp [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  /// TODO this is not yet working
+  // Stop the Holoscan app
+  holoscan_app->executor().interrupt();
+  // Wait for the Holoscan app to stop
+  future.wait();
+
+  return ret;
+}
+
+#include "main.moc"
```
[the remark this answers is not in the document]
Comment (AndreasHeumann) [reply applications/qt/main.cpp]: Thanks, fixed after running lint and getting the errors about missing blank lines.

File: applications/qt/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Qt integration
+
+This application demonstrates how to integrate a Qt application with Holoscan.
+
+The application uses the `QtVideOp` operator to display a video stream in a Qt application.
```
[the remark this answers is not in the document]
Comment (AndreasHeumann) [reply applications/qt/README.md:5]: Thanks, fixed the QtVideoOp. But officially it should be Qt with the lower case `t`.

File: applications/CMakeLists.txt [hunk 3]
```diff
@@ -69,6 +69,8 @@ add_holohub_application(object_detection_torch HOLOSCAN_SAMPLE_APP)
 
 add_holohub_application(prohawk_video_replayer DEPENDS OPERATORS prohawk_video_processing)
 
+add_holohub_application(qt DEPENDS OPERATORS qt_video npp_filter)
```
Comment (jjomier) [inline applications/CMakeLists.txt:72]: We need to have an explicit name for the application. "qt_video_replayer" ?

File: applications/qt/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        software-properties-common="0.99.*" \
+      && add-apt-repository ppa:okirby/qt6-backports -y; \
+    fi \
+    && apt update \
+    && apt install --no-install-recommends -y \
+        qt6-base-dev \
+        qt6-declarative-dev \
+        qml6-module-qtquick-layouts \
+        qml6-module-qtquick-controls \
+        qml6-module-qtquick-templates \
```
Comment (jjomier) [inline applications/qt/Dockerfile:42]:
```suggestion
        qml6-module-qtquick-templates \
        qml6-module-qtquick-window \
```

Comment (AndreasHeumann) [reply applications/qt/Dockerfile]: Thanks, added qml6-module-qtquick-window and tested.

File: applications/CMakeLists.txt (hunk 3 above)
Comment (AndreasHeumann) [reply applications/CMakeLists.txt]: Good idea, change the name.

**Review by jjomier — APPROVED** [review]
LGTM

**Comment by AndreasHeumann** [comment]
@jjomier Thanks, could you merge?

**Pull request merged**